actors/stage/src/main/java/com/ea/orbit/actors/cluster/JGroupsClusterPeer.java

```diff
@@ -144,13 +145,21 @@ public Address call()
                         udp.setMulticastPort(udp.getMulticastPort() + ((clusterName.hashCode() & 0x8fff_ffff) % portRangeLength));
                     }
 
+
+                    ProtocolStack stack = baseChannel.getProtocolStack();
+                    FORK fork = (FORK) stack.findProtocol(FORK.class);
+                    if (fork == null)
+                    {
+                        stack.insertProtocol(fork = new FORK(), ProtocolStack.ABOVE, FRAG2.class);
```

Comment:
Now I see why you added it, must be a regression in JGroups..

Caused by: java.lang.NullPointerException
	at org.jgroups.protocols.FORK.createForkStack(FORK.java:199)
	at org.jgroups.fork.ForkChannel.<init>(ForkChannel.java:79)
	at com.ea.orbit.actors.cluster.JGroupsClusterPeer$1.call(JGroupsClusterPeer.java:148)
	... 7 more